aioredis/commands/geo.py

```diff
@@ -0,0 +1,155 @@
+from aioredis.util import wait_convert, wait_convert_with_opts, _NOTSET
+
+
+class GeoCommandsMixin:
+    """Geo commands mixin.
+
+    For commands details see: http://redis.io/commands#geo
+    """
+
+    def geoadd(self, key, longitude, latitude, member, *args, **kwargs):
+        """Add one or more geospatial items in the geospatial index represented
+        using a sorted set
+        """
+        return self._conn.execute(
+            b'GEOADD', key, longitude, latitude, member, *args, **kwargs
+        )
+
+    def geohash(self, key, member, *args, **kwargs):
+        """Returns members of a geospatial index as standard geohash strings
+        """
+        encoding = _NOTSET
+        if 'encoding' in kwargs:
+            encoding = kwargs.pop('encoding')
```

Comment:
No need in such `encoding` parsing, it will be extracted from `kwargs` in `execute` method.